devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy

```diff
@@ -195,11 +206,24 @@ def refresh() {
 	delayBetween(commands,100)
 }
 
+def healthPoll() {
+	def cmds = refresh()
+	cmds.each{ sendHubCommand(new physicalgraph.device.HubAction(it))}
+}
+
+def configure() {
+	unschedule()
+	schedule("0 0/5 * * * ? *", "healthPoll")
```

Comment:
The DTH scheduling method `runEvery5Minutes()` should be going out to data-mgmt production this Wed. 9/29. Let's test this method in dev now and have one PR to merge rather than 2. 
https://smartthings.atlassian.net/wiki/display/DATAMGMT/2016/09/26/2016-09-29+Release